conjure-java-core/src/test/java/com/palantir/conjure/java/types/AliasTests.java

```diff
@@ -43,4 +48,25 @@ public void testValueOf_external() {
     public void testValueOf_externalNested() {
         assertThat(ExternalLongAliasTwo.valueOf("3")).isEqualTo(ExternalLongAliasTwo.of(ExternalLongAliasOne.of(3L)));
     }
+
+    @Test
+    public void testValueOf_largeDouble_lossyDouble() throws JsonProcessingException {
+        // So large it doesn't fit in a double; jackson silently drops precision
+        assertThat(TEST_MAPPER.readValue("9007199254740992.234", DoubleAliasExample.class))
+                .isEqualTo(DoubleAliasExample.of(9007199254740992.0));
```

Comment:
To clarify, this is what I meant when I said Jackson is lossy; I'm not sure why we'd accept `9007199254740993.0` but reject `9007199254740993` (the number cannot be represented in `double`)